mobile-widgets/qmlmanager.cpp

```diff
@@ -444,7 +444,7 @@ void QMLManager::finishSetup()
 {
 	// Initialize cloud credentials.
 	git_local_only = !prefs.cloud_auto_sync;
-	QMLPrefs::instance()->setCredentialStatus((qPrefCloudStorage::cloud_status) prefs.cloud_verification_status);
+
```

Comment:
the commit message refers to `the "if" will not execute` which I'm sure made sense when you wrote the series and thought about the changes, but is really hard to understand when you read this commit. What `if` and where?
Would it be better to describe in words what cannot happen?

Reply:
Refrased